### DISCUSSION TITLE
SubWorkflow From Code Sample

### DISCUSSION THREAD

**Discussion by Eatric** [elsa-workflows/elsa-core#4417] [Q&A]
Good afternoon, can you please advise me if there is any example implementation of calling a child Workflow from code?

**Comment by sfmskywalker** [comment]
Hello @Eatric , sure thing. I just created and pushed an example that you can see [here](https://github.com/elsa-workflows/elsa-core/tree/v3/src/samples/aspnet/Elsa.Samples.AspNet.ChildWorkflows/Workflows).
